frontend/Dockerfile

```diff
@@ -14,4 +14,4 @@ EXPOSE 8081
 RUN yarn
 RUN yarn run build:prod
 
-CMD ["yarn", "run", "start:prod"]
```

Comment:
prod takes 6 seconds vs 3.5 seconds as far as I can tell, if we want to use not-prod for local dev then we should maybe have an override? the main difference is that non-prod doesn't have uglify. seems to be the main culprit. We could also try removing sourceMap:true for uglify